cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaUpgradeDowngradeMockTest.java

```diff
@@ -205,21 +211,29 @@ public void testUpgradeWithoutMessageAndProtocolVersions(VertxTestContext contex
         initialize(initialKafka)
                 .onComplete(context.succeeding(v -> {
                     context.verify(() -> {
+                        KafkaStatus status = Crds.kafkaOperation(client).inNamespace(NAMESPACE).withName(CLUSTER_NAME).get()
+                                .getStatus();
+                        assertThat(status.getKafkaVersion(), is(KafkaVersionTestUtils.PREVIOUS_KAFKA_VERSION));
+                        assertThat(status.getOperatorLastSuccessfulVersion(), is(KafkaAssemblyOperator.OPERATOR_VERSION));
```

Comment:
You seem to repeat this code in every test. Should it have some method similar to `assertVersionsInStrimziPodSet`?

Reply:
Sure this seems like a good idea. will update the PR.